### DISCUSSION TITLE
About sensitivity analysis output

### DISCUSSION THREAD

**Discussion by Sawy25** [Open-Systems-Pharmacology/Forum#1541] [Questions & Problems] [answered]
Hi 

I read the pk-sim manual about the meaning of the output of sensitivity analysis, but I am still confused about it. When we get a value of 0.5 for instance according to the manual, it means that upon increasing the parameter value by 10% the pk metric changes by 5%. My confusion is about the direction of change, the default values for variation range, and number of steps will allow a 10% increase and decrease of the parameter value, so why do we interpret the result as a 10% increase, not a decrease, in parameter value? we may actually get non-symmetric results for the change in the pk metric when we increase/decrease the parameter.

Can someone kindly clarify more about this?

Thanks

**Comment by StephanSchaller** [comment]
https://docs.open-systems-pharmacology.org/shared-tools-and-example-workflows/sensitivity-analysis

--> Mathematical Background

This calculation is made for each step:
![image](https://github.com/Open-Systems-Pharmacology/Forum/assets/25477643/e7786c66-22cb-44e4-b2b6-2efb52dd6084)

And then the average is calculated.
So if you decrease also your output will decrease if you have a positive sensitivity.

Best,
Stephan

**Reply by Sawy25** [reply]
Thanks a lot!
So when you say the average is calculated, does it mean that if the magnitude of change in my output is the not same when I change the parameter (just in a different direction, for instance, 30%,20%), then the output will be 25% or 2.5?  so we can not know which direction has the bigger influence.

**Comment by PavelBal** [comment, marked as answer]
@Sawy25 You can use the R package `{esqlabsR}` https://github.com/esqLABS/esqlabsR
to perform a more sophisticated sensitivity analysis as described here: https://esqlabs.github.io/esqlabsR/dev/articles/sensitivity-analysis.html

With the package, you can produce sensitivity calculations for a range of parameter values to identify non-linear sensitivities with figures like this:

![](https://esqlabs.github.io/esqlabsR/dev/articles/sensitivity-analysis_files/figure-html/spider-plot-1.png)

**Reply by Sawy25** [reply]
That is so handy and clear. Many thanks!
